arbitrum/config.go

```diff
@@ -37,6 +37,11 @@ type Config struct {
 	ClassicRedirect        string        `koanf:"classic-redirect"`
 	ClassicRedirectTimeout time.Duration `koanf:"classic-redirect-timeout"`
 	MaxRecreateStateDepth  int64         `koanf:"max-recreate-state-depth"`
+
+	// Espresso specific flags
+	Espresso          bool   `koanf:"espresso"`
+	HotShotUrl        string `koanf:"hotshot-url"`
```

Comment:
This is a bit confusing for me. As `HotShot` is our protocol of consensus and it doesn't seem necessary for people knowing anything about `HotShot` in the integration. Maybe `EspressoUrl` is more appropriate?

Reply:
Yeah I think you have a point, although we seem to have developed a convention for referring to "HotShot" for all things consensus (we have variable names like hotshot_contract, hotshot_url, etc. sprinkled all over the place). This is definitely something to think about as we start making more client facing tools like SDKs. I personally think its OK to refer to HotShot directly for now.